### ISSUE TITLE
toJavaProto/fromJavaProto is incomplete for FieldOptions on UnknownFieldSet

### ISSUE THREAD

**Issue by mantasindrasius** [scalapb/ScalaPB#934]
Conversions from java protobuf library to scala and back are missing the 

Misses UnknownFieldSet in one of the fields:
```scala
FileDescriptorProto.fromJavaProto(javaFileDescriptorProto)
```

This works:
```scala
FileDescriptorProto.parseFrom(javaFileDescriptorProto.toByteArray)
```

The methods seems lacking this field:
https://github.com/scalapb/ScalaPB/blob/master/scalapb-runtime/src/main/scalajvm/com/google/protobuf/descriptor/FieldOptions.scala#L211

**Comment by thesamet** [comment]
Yes, I confirm that unknown fields are not converted to/from java for any ScalaPB message. Will you be able to work on a PR for this?

**Comment by mantasindrasius** [comment]
Not sure at the moment, but I'll notify in case I'll be able to.

**Comment by thesamet** [comment]
@mantasindrasius That's ok. For context, does this issue has practical implications on your project or is it just a gap that you've identified?
